apps/service_providers/llm_service/prompt_context.py

```diff
@@ -38,6 +40,23 @@ def get_source_material(self):
         except SourceMaterial.DoesNotExist:
             return ""
 
+    def get_media_summaries(self):
+        """
+        Example output:
+        File (27): summary1
+        ----------
+        File (28): summary2
+        ----------
```

Comment:
more compact would be to use markdown list notation:
```suggestion
        * File (27): summary1
        * File (28): summary2
```

If multiline summaries are an issue we could use yaml or json

Reply:
Might also want to include the content type so that it knows if it's an image or not.